pgjdbc/src/test/java/com/yugabyte/FallbackOptionsLBTest.java

```diff
@@ -0,0 +1,139 @@
+package com.yugabyte;
+
+import org.postgresql.util.PSQLException;
+
+import com.yugabyte.ysql.LoadBalanceProperties;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.sql.*;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class FallbackOptionsLBTest {
+  private static int numConnections = 12;
+  private static String path;
+
+  public static void main(String[] args) throws SQLException, ClassNotFoundException {
+    // Start YugabyteDB cluster if env YBDB_PATH is defined. Else, assume that the cluster is up.
+    startYBDBCluster();
+    try {
+      Class.forName("org.postgresql.Driver");
+      String url1 = "jdbc:yugabytedb://localhost:5433/yugabyte?load-balance=true&" + LoadBalanceProperties.TOPOLOGY_AWARE_PROPERTY_KEY + "=";
+      String url2 = "aws.us-west.us-west-2a,aws.us-west.us-west-2b:1,aws.us-west.us-west-2c:2";
+      Connection conn = DriverManager.getConnection(url1 + url2,
+          "yugabyte", "yugabyte");
+      Statement stmt = conn.createStatement();
+      stmt.execute("CREATE TABLE IF NOT EXISTS employee" +
+          "  (id int primary key, name varchar, age int, language text)");
+      System.out.println("Created table");
+      conn.close();
+
+      // All valid/available placement zones
+      createConnections(url1 + "aws.us-west.us-west-2a,aws.us-west.us-west-2c", 6, 0, 6);
+      createConnections(url1 + "aws.us-west.us-west-2a,aws.us-west.us-west-2b:1,aws.us-west.us-west-2c:2", 6, 6, 0);
+      createConnections(url1 + "aws.us-west.us-west-2a:1,aws.us-west.us-west-2b:2,aws.us-west.us-west-2c:3", 12, 0, 0);
+      createConnections(url1 + "aws.us-west.*,aws.us-west.us-west-2b:1,aws.us-west.us-west-2c:2", 4, 4, 4);
```

Comment:
`aws.us-west.*,aws.us-west.us-west-2b:1,aws.us-west.us-west-2c:2` is this a valid test? 

Does this mean all the nodes in `us-west` will be primary or `*`  will overridden by the subsequent `us-west-2b:1` and `us-west-2c:2`?

Reply:
Yes, we do not check if the placement values overlap with each other. So first primary placement values will be checked/honoured which is all zones (`*`) in `us-west`.
You are right, this particular test is not an appropriate case but I've added it just for testing.